server/data/nl/danielsjf.txt

```diff
@@ -2871,3 +2871,507 @@ Onze droger is alweer stuk.
 Bij de rode lichten ga je rechtdoor.
 Geen zorgen, ik heb een vouwfiets bij me.
 Je mag niet zomaar binnen in de milieuzone.
+Ik kon er de ironie wel van inzien.
+De straaljager had nog net voldoende brandstof om  veilig op het vliegdekschip te landen. 
+De medicijnen maakten haar paranoia. 
+Hij was een echte vakman.
+Zij werkt met veel liefde voor haar vak.
+De juffrouw en de meester bereidden altijd samen hun lessen voor.
+Hij kon zijn emoties niet de baas.
+De directrice was streng maar rechtvaardig.
+Hij moest nu beslissen hoe hij verder wou met zijn leven.
+Enkel de waarheid telt.
+Zij heeft een stevige ruggengraat.
+Hij had veel haar op zijn tanden. 
+Vroeger was alles beter.
+Ik verdoen mijn tijd aan het antwoorden op zijn vragen.
```

Comment:
"Verdoen.. aan" => "verdoe .. met"
"antwoorden op" => "beantwoorden van" (BE/NL?)